### DISCUSSION TITLE
Autocomplete Errors

### DISCUSSION THREAD

**Discussion by rhurlbatt** [anvilistas/anvil-extras#423] [Q&A]
I am not sure if this is a bug or simply the way the form code is being handled.

I have two errors showing up in my logs both to do with the same line of code:
```
    def _on_hide(self, **e_args):
        """This method is called when the TextBox is removed from the screen"""
        self._lp_node.remove()
        _S(_window).off("resize", self._reset_position)
```

```
ExternalError: NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
at app/anvil_extras/Autocomplete/__init__.py, line 221
```
I don't have browser info for this one.

```
ExternalError: NotFoundError: The object can not be found here.
at app/anvil_extras/Autocomplete/__init__.py, line 221
```
Browser Info: Mozilla/5.0 (iPhone; CPU iPhone OS 16_4_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.4 Mobile/15E148 Safari/604.1

I have a hunch that this is occurring when the form is part way through some process and then the form is unloaded but I cannot recreate it. Any ideas on how to handle this?

**Comment by s-cork** [comment]
Probably just wrapping that line in a try except will be enough
